CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_unified_rag v0.1.0-a3
+[PR #9](https://github.com/fivetran/dbt_unified_rag/pull/9) includes the following updates: 
+
+## Bug Fix
```

Comment:
Since this is going to be updating the underlying data flowing into the incremental model, we should callout that a full refresh is required when making this upgrade.

Reply:
Updated.